my_game.py

```diff
@@ -208,6 +203,10 @@ def on_update(self, delta_time):
         # Update the player shots
         self.player_shot_list.update()
 
+        if self.up_pressed:
+            if not self.player_sprite.speed > self.max_speed:
+                self.player_sprite.speed += 0.05
```

Comment:
The 0.05 need to go in to a variable. Let's make a global variable PLAYER_THRUST, and use that here.

At beginning of file:
PLAYER_THRUST = 0.05

Then use that variable here.